libpod/container.go

```diff
@@ -157,6 +157,11 @@ func (c *Container) State() (ContainerState, error) {
 	return c.state.State, nil
 }
 
+// MountPoint returns the mount point of the continer
+func (c *Container) MountPoint() string {
+	return c.state.Mountpoint
```

Comment:
Can you add c.lock.Lock()/c.lock.Unlock() and c.runtime.state.UpdateContainer(c) here - this is volatile data, so we need to update it from the database before returning it